cli/util/upload_state.go

```diff
@@ -143,6 +144,9 @@ loop:
 				object.NewAttribute("BlockTime", strconv.FormatUint(h.Timestamp, 10)),
 			}
 		)
+		if len(stateRoot.Witness) > 0 {
+			attrs = append(attrs, object.NewAttribute(neofs.DefaultWitnessAttribute, hex.EncodeToString(stateRoot.Witness[0].Bytes())))
```

Comment:
btw, base64 is more compact. I doubt we specifically need  hex here since it's not likely we'll be using this attribute for matching of any kind.